Cargo.toml

```diff
@@ -141,7 +141,7 @@ sentry-core = "0.32.2"
 sentry-kafka-schemas = { version = "0.1.86", default-features = false }
 sentry-release-parser = { version = "1.3.2", default-features = false }
 sentry-types = "0.32.2"
-sentry_usage_accountant = { version = "0.1.0", default-features = false }
+sentry_usage_accountant = { git = "https://github.com/getsentry/rust-usage-accountant.git", rev = "48df2ef", default-features = false }
```

Comment:
What's the reason for this upgrade? Can we ask someone to release the usage accountant crate?

Reply:
It's still a PR which changes the callback to emit a `Message` instead of `Vec<u8>`